zrml/court/src/lib.rs

```diff
@@ -260,54 +392,43 @@ mod pallet {
     /// For each market, this holds the dispute information for each dispute that's
     /// been issued.
     #[pallet::storage]
-    #[pallet::getter(fn disputes)]
     pub type Disputes<T: Config> = StorageMap<
         _,
         Blake2_128Concat,
         T::MarketId,
         Vec<MarketDispute<T::AccountId, T::BlockNumber>>,
-        ValueQuery,
     >;
 
     /// A mapping of market identifiers to the block they were disputed at.
     /// A market only ends up here if it was disputed.
     #[pallet::storage]
-    #[pallet::getter(fn market_ids_per_dispute_block)]
     pub type MarketIdsPerDisputeBlock<T: Config> =
-        StorageMap<_, Blake2_128Concat, T::BlockNumber, Vec<T::MarketId>, ValueQuery>;
+        StorageMap<_, Blake2_128Concat, T::BlockNumber, Vec<T::MarketId>>;
 
     /// A mapping of market identifiers to the block that they were reported on.
     #[pallet::storage]
-    #[pallet::getter(fn market_ids_per_report_block)]
     pub type MarketIdsPerReportBlock<T: Config> =
-        StorageMap<_, Blake2_128Concat, T::BlockNumber, Vec<T::MarketId>, ValueQuery>;
+        StorageMap<_, Blake2_128Concat, T::BlockNumber, Vec<T::MarketId>>;
 
     #[pallet::storage]
-    #[pallet::getter(fn market_to_swap_pool)]
-    pub type MarketToSwapPool<T: Config> =
-        StorageMap<_, Blake2_128Concat, T::MarketId, Option<PoolId>, ValueQuery>;
+    pub type MarketToSwapPool<T: Config> = StorageMap<_, Blake2_128Concat, T::MarketId, PoolId>;
 
     /// For each market, this holds the dispute information for each dispute that's
     /// been issued.
     #[pallet::storage]
-    #[pallet::getter(fn markets)]
-    pub type Markets<T: Config> = StorageMap<
-        _,
-        Blake2_128Concat,
-        T::MarketId,
-        Option<Market<T::AccountId, T::BlockNumber>>,
-        ValueQuery,
-    >;
```

Comment:
Optional queries are being used because, IIRC, it is impossible to distinguish between `vec![]` being a stored value without results or `vec![]` being the returned value of an non-existent query where `T: Default`.